app/controllers/github_webhooks_controller.rb

```diff
@@ -0,0 +1,58 @@
+class GithubWebhooksController < ActionController::Base
+  skip_before_action :verify_authenticity_token
```

Comment:
Could to explain to me why you are skipping this action please? =)
Shouldn't it be `authenticate_github_request!` ? I mean if it's from the `github_webhook` gem?

Reply:
The authenticate_github_request method is just to validade the webhook_secret of the github, it should match with the GITHUB_WEBHOOK_SECRET env var that is required to run the integration. If it is not a match, an error is raised.

The json request from github webhook doesn't have the CSRF token that is required to validate the authenticity, so this is a workaround to avoid rails to raise ActionController::InvalidAuthenticityToken in order the integration to works properly.

Reply:
Oh yeah, totally forgot about the `CSRF token`. Good plan 👍